### PR TITLE
story2task2 - Setup the BookDetail component

### PR DESCRIPTION
**Pull request by GDTimi**
The BookDetail component is in place, with placeholder ID in use. After we have put this component through PR process, we can do story1task1 to thread the data through from the AllBooks component (after story1task2 and story1task3 are through also).

### CODE REVIEW

File: src/components/BookDetail/BookDetail.jsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    fetch(`https://book-swap-api.dev.io-academy.uk/api/books/${bookID}`, {
+        mode: 'cors',
+        headers: {
+            "Content-Type": "application/json",
+            "Accept": "application/json"
+        },
+        method: "GET",
+    }).then(res => res.json())
+        .then(bookData => {
+            console.log(bookData.data)
```
Comment (Andrei-Trifu) [inline src/components/BookDetail/BookDetail.jsx]: Can we remove this console.log?

File: src/components/BookDetail/BookDetail.jsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+            setBookImage(bookData.data.image);
+            setBookTitle(bookData.data.title);
+            setBookAuthor(bookData.data.author);
+            setBookYear(bookData.data.year);
+            setBookPageCount(bookData.data.page_count);
+            setBookGenre(bookData.data.genre.name);
+            setBookBlurb(bookData.data.blurb);
+            // console.log(bookAuthor)
```
Comment (Andrei-Trifu) [inline src/components/BookDetail/BookDetail.jsx]: if this is not used can we also remove it?

File: src/main.jsx [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 import ReactDOM from 'react-dom/client'
 import App from './App.jsx'
-import './index.css'
+//import './index.css'
```
Comment (Andrei-Trifu) [inline src/main.jsx]: remove this if not needed

File: src/components/BookDetail/BookDetail.jsx [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+            <div>
+                <h1>{bookTitle}</h1>
+                <p>{bookAuthor}</p>
+                <p>{bookYear}</p>
+                <p>{bookPageCount}</p>
+                <p>{bookGenre}</p>
+                <p>{bookBlurb}</p>
+            </div>
+
```
Comment (Andrei-Trifu) [inline src/components/BookDetail/BookDetail.jsx]: maybe remove extra spaces

File: src/components/BookDetail/BookDetail.jsx [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+        })
+
+    }, [])
+
+    return (
+        <>
+        <h1>Book Swap</h1>
+        <div className="detailContainer">
+            <div>
+                <img className="detailContainerImage" src={bookImage} alt={bookTitle} />
```
Comment (Andrei-Trifu) [inline src/components/BookDetail/BookDetail.jsx]: why are you using camel case for css classes instead of "detail-container-image"?

**Review by Andrei-Trifu — CHANGES_REQUESTED** [review]

File: src/components/BookDetail/BookDetail.jsx (hunk 1 above)
Comment (GDTimi) [reply src/components/BookDetail/BookDetail.jsx]: Done

File: src/components/BookDetail/BookDetail.jsx (hunk 2 above)
Comment (GDTimi) [reply src/components/BookDetail/BookDetail.jsx]: Yep, done

File: src/main.jsx (hunk 3 above)
Comment (GDTimi) [reply src/main.jsx]: Yep, removed.

File: src/components/BookDetail/BookDetail.jsx (hunk 4 above)
Comment (GDTimi) [reply src/components/BookDetail/BookDetail.jsx]: Done. :)

File: src/components/BookDetail/BookDetail.jsx (hunk 5 above)
Comment (GDTimi) [reply src/components/BookDetail/BookDetail.jsx]: Ok - class names have been updated.

**Pull request merged**